lib/common/mobile/android/logcat-helper.ts

```diff
@@ -101,6 +87,20 @@ export class LogcatHelper implements Mobile.ILogcatHelper {
 			delete this.mapDevicesLoggingData[deviceIdentifier];
 		}
 	}
+
+	private async getLogcatStream(deviceIdentifier: string, pid?: string) {
+		const device = await this.$devicesService.getDevice(deviceIdentifier);
+		const minAndroidWithLogcatPidSupport = "7.0.0";
+		const isLogcatPidSupported = !!device.deviceInfo.version && semver.gte(semver.coerce(device.deviceInfo.version), minAndroidWithLogcatPidSupport);
```

Comment:
what if we do not have device? It will still fail maybe? Or if the deviceInfo is null ?

Reply:
I was wondering about the same problem.

Actually when `adb devices` returns some data - for example `emulator-5554 device`, we know for sure in this moment the device's identifier and device's status. In other words we should have at least 
```
deviceInfo: {
    identifier: "emulator-5554",
    status: "connected"
}
``` 
So it is not possible to be here with `deviceInfo` or `device` with value `null`.